examples/creatives/testers_nonlinear.js

```diff
@@ -40,10 +40,10 @@ class TestersNonLinear extends BaseSimidCreative {
     creativeDimensions.height = document.getElementById('resize_height').value;
 
     const resizeParams = {
-      mediaDimensions: {},
+      videoDimensions: this.environmentData.videoDimensions,
       creativeDimensions: creativeDimensions
     };
 
-    this.simidProtocol.sendMessage(CreativeMessage.REQUEST_RESIZE, resizeParams);
+    super.requestResize(resizeParams);
```

Comment:
Since TestersNonlinear doesn't override requestResize, you don't need to use super here, you can use `this.requestResize`

Reply:
fixed